lumicks/pylake/force_calibration/power_spectrum_calibration.py

```diff
@@ -84,7 +84,9 @@ def generate_table(entries):
                 [
                     key,
                     f"{param.description}{f' ({param.unit})' if param.unit else ''}",
-                    param.value if isinstance(param.value, str) else f"{param.value:.6g}",
+                    param.value
+                    if isinstance(param.value, str)
+                    else ("" if param.value is None else f"{param.value:.6g}"),
```

Comment:
The nested ternaries look confusing, but I have no better suggestion.
Probably pattern matching would be the best option, but we would need python 3.10.